config/test.exs

```diff
@@ -12,3 +12,9 @@ config :erlef, Erlef.Mailer, adapter: Swoosh.Adapters.Test
 
 # Print only warnings and errors during test
 config :logger, level: :warn
+
+config :erlef, Erlef.Data.Repo,
+  database: "erlef_website_test",
+  pool: Ecto.Adapters.SQL.Sandbox,
+  migration_primary_key: [id: :uuid, type: :binary_id],
```

Comment:
Are we sure we want UUID for primary keys? I'm not sure the database will ever be big enough or we will have a situation where we can't reliable increment an integer PK... but I'm all ears for the argument.

Reply:
i usually opt for UUID from the start. I've been bitten enough over the years using integers for primary keys. The uniqueness factor plays a big part in that. Plus, the effort is minimal to implement from the start and far more difficult to implement later.

Reply:
👍